Sources/ProjectSpec/TargetSource.swift

```diff
@@ -119,6 +133,7 @@ public struct TargetSource: Equatable {
         self.type = type
         self.optional = optional
         self.buildPhase = buildPhase
+        self.headerVisibility = headerVisibility
```

Comment:
I noticed this wasn't being properly set. Didn't necessarily intend to commit it with this.

Reply:
I'm ok with it being here